### DISCUSSION TITLE
How to change default model downloading path?

### DISCUSSION THREAD

**Discussion by hongbo-miao** [roboflow/supervision#377] [Q&A] [answered]
This code will download `yolov8s.pt` model in currently folder. Is there a way to change the path? Thanks!

```python
from ultralytics import YOLO

model = YOLO("yolov8s.pt")
```

**Comment by onuralpszr** [comment, marked as answer]
Just type path and it will download

```python
model = YOLO("model/yolov8s.pt")
```

**Reply by hongbo-miao** [reply]
Thanks @onuralpszr ! It works! 😃
I thought if provide custom path, it means use custom model (I think I read somewhere, but I cannot find that doc any more... maybe I am wrong), which is a little confusing to me.

**Reply by onuralpszr** [reply]
@hongbo-miao  you're welcome :)  First I thought the same then I figure out that also works :)

**Reply by SkalskiP** [reply]
@onuralpszr, thanks for the support!
